### PR TITLE
version fix for doc builds and tox test changes

### PR DESCRIPTION
**Pull request by nabobalis**
Just a test.

### CODE REVIEW

**Comment by nabobalis** [comment]
I think it is safe for me to make this with my 30 commits, got to increase my chart placement.

File: setup.cfg [hunk 1]
```diff
@@ -50,14 +50,16 @@ docs =
...
+sunpy.data = sunpyrc, test/*, test/*/*
 sunpy.database = tests/test_table.txt
 sunpy.io = special/asdf/schemas/sunpy.org/sunpy/*/*.yaml, special/asdf/schemas/sunpy.org/sunpy/*/*/*.yaml
 sunpy.tests = figure_hashes_py36.json
 
 [options.entry_points]
 asdf_extensions =
     sunpy = sunpy.io.special.asdf.extension:SunpyExtension
+pytest11 =
+    asdf = asdf.tests.schema_tester
```
Comment (Cadair) [inline setup.cfg]: has this made it so you have to have asdf installed to run our tests?

File: sunpy/net/dataretriever/tests/test_noaa.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import tempfile
```
Comment (Cadair) [inline sunpy/net/dataretriever/tests/test_noaa.py]: not needed?

File: tox.ini [hunk 3]
```diff
@@ -1,53 +1,36 @@
...
-    build_docs: ruamel.yaml
-    build_docs: sphinx
-    build_docs: sphinx-astropy
-    build_docs: sunpy-sphinx-theme
-    build_docs: git+git://github.com/hawkowl/towncrier
 commands =
-    offline,astropydev,numpydev: pytest -m "not figure" --cov=./sunpy {posargs}
-    online: pytest --timeout=500 -m "not figure" --remote-data=any --cov=./sunpy {posargs}
-    build_docs: sphinx-build docs docs/_build/html -W -b html
+    offline,astropydev,numpydev: pytest --verbose --pyargs sunpy -m "not figure" --cov=sunpy --rootdir={toxinidir} --junitxml=junit/test-results.xml --cov-report=xml --cov-report=html {toxinidir}/docs
```
Comment (Cadair) [inline tox.ini]: needs `{postargs}`?

File: tox.ini [hunk 4]
```diff
@@ -58,13 +41,12 @@ conda_deps =
     pandas = 0.23.4
     pytest-cov
 conda_channels = conda-forge
-commands = pytest -m "figure" --cov=./sunpy
+commands = pytest -v --pyargs sunpy -m "figure" --cov=sunpy --junitxml=junit/test-results.xml --cov-report=xml --cov-report=html
```
Comment (Cadair) [inline tox.ini]: postargs here as well?

File: tox.ini [hunk 5]
```diff
@@ -91,4 +73,4 @@ conda_deps =
     zeep
 conda_channels = conda-forge
 commands =
-    pytest -m "not figure" --cov=./sunpy {posargs}
+    pytest -v --pyargs sunpy -m "not figure" --cov=sunpy --junitxml=junit/test-results.xml --cov-report=xml --cov-report=html {posargs}
```
Comment (Cadair) [inline tox.ini]: I am not a fan of putting the junit and the xml cov report in the tox.ini I think they are better off being a thing specific to Azure?

File: setup.cfg (hunk 1 above)
Comment (nabobalis) [reply setup.cfg]: Yes

File: tox.ini (hunk 5 above)
Comment (nabobalis) [reply tox.ini]: K

File: tox.ini (hunk 4 above)
Comment (nabobalis) [reply tox.ini]: K

File: tox.ini (hunk 3 above)
Comment (nabobalis) [reply tox.ini]: K

File: sunpy/net/dataretriever/tests/test_noaa.py (hunk 2 above)
Comment (nabobalis) [reply sunpy/net/dataretriever/tests/test_noaa.py]: K

File: setup.cfg (hunk 1 above)
Comment (Cadair) [reply setup.cfg]: Not the end of the world I suppose.

**Pull request merged**